elements/pfe-select/src/pfe-select.scss

```diff
@@ -0,0 +1,9 @@
+@import "../../pfe-sass/pfe-sass";
+
+:host {
+  display: block;
+}
+
+:host([hidden]) {
+  display: none;
+}
```

Comment:
@markcaron @diwanshi the lightdom CSS file is temporary; as soon as Edge releases the latest version (with Chromium under the hood) and we drop support for IE11, we can stop loading the lightdom.css files entirely. 

Can we move most of those styles back into the original web component sass file?

Reply:
Unless I'm misunderstanding...We want light DOM support still. PFE Select is intended to style all standard selects, including forms that may or may not require JavaScript.

Or is everyone else agreeing that all non-JS support should look like browser-chromed inputs? I, personally, feel that we should try to ship as much as possible without JS. I know this means there's an extra include (until we get a service to do all this for the Dev).

Reply:
Hi @markcaron 

I moved back all the styles in lightdom file to this file. Can you please review if everything is okay. I tested this and it works fine.